### ISSUE TITLE
ARIA6 and others: Bad links

### ISSUE THREAD

**Issue by yatil** [w3c/wcag#317]
In technique ARIA6, and presumably others, there are bad links to ARIA and adjacent specs:

> Authors should be aware that `aria-label` may be disregarded by assistive technologies in situations where `aria-labelledby` is used for the same object. For more information on the naming hierarchy please consult the [ARIA specification](http://www.w3.org/TR/wai-aria/roles#textalternativecomputation) and the [accessible name and description calculation](http://www.w3.org/TR/html-aapi/#accessible-name-and-description-calculation) in the HTML to Platform Accessibility APIs Implementation Guide. Authors should be aware that use of `aria-label` will override any native naming such as `alt` on images or `label` associated with a form field using the `for` attribute.

The first link (http://www.w3.org/TR/wai-aria/roles#textalternativecomputation) goes to a 404 and should be either replaced with https://www.w3.org/TR/wai-aria/#textalternativecomputation or (IMHO better) https://www.w3.org/TR/accname-1.1/#mapping_additional_nd_te

The second link (http://www.w3.org/TR/html-aapi/#accessible-name-and-description-calculation) goes to a document that is marked “no longer in active maintenance” and might be replaced with https://www.w3.org/TR/accname-1.1/#mapping_additional_nd

You might also want to consider to change the sentence so only one link is needed now that both calculations can be found in one document.

(Thanks to @boutrosabichedid who made me aware of this issue in https://github.com/w3c/wai-wcag-quickref/issues/157.)

**Comment by fstrr** [comment]
@michael-n-cooper

Hi, Michael.

The first link only appears in the ARIA6 document, which is easy. The second link appears in 11 xml files:
1. ARIA1.xml
2. ARIA6.xml
3. ARIA7.xml
4. ARIA8.xml
5. ARIA9.xml
6. ARIA10.xml
7. ARIA13.xml
8. ARIA14.xml
9. ARIA15.xml
10. ARIA16.xml
11. F65.xml

Are those XML files safe to update or should they be left alone for you to do?

**Comment by alastc** [comment]
If this is only in the XML files (but is fixed in the WCAG 2.1 or the latest editors draft) this issue can be closed.
